build.py

```diff
@@ -2,6 +2,12 @@
 import os, sys
 import platform
 
+TOP_DIR = os.getcwd()
```

Comment:
Is it intended that the TOP_DIR will be different depending on where the script is called from? 

If not, it might be safer to use `os.path.abspath( os.path.dirname( __file__ ) )`

Reply:
No. It's only supposed to run in the same directory as PythonMonkey's `pyproject.toml`

Reply:
```suggestion
TOP_DIR = os.path.abspath(os.path.dirname(__file__))
```